packages/ui/components/button/Button.tsx

```diff
@@ -176,11 +179,23 @@ export const Button = forwardRef<HTMLAnchorElement | HTMLButtonElement, ButtonPr
     },
     <>
       {StartIcon && (
-        <StartIcon
-          className={classNames("inline-flex", size === "icon" ? "h-4 w-4 " : "mr-2 h-4 w-4 stroke-[1.5px]")}
-        />
+        <>
+          {size === "fab" ? (
+            <>
+              <StartIcon className="mr-2 hidden h-4 w-4 stroke-[1.5px] sm:inline-flex" />
+              <Icon.FiPlus className="inline h-6 w-6 sm:hidden" />
+            </>
+          ) : (
+            <StartIcon
+              className={classNames(
+                "inline-flex",
+                size === "icon" ? "h-4 w-4 " : "mr-2 h-4 w-4 stroke-[1.5px]"
+              )}
+            />
+          )}
+        </>
```

Comment:
unfortunately sometimes we have a CTA with a StartIcon and sometimes with an EndIcon so we need to apply the FAB-magic on both

(on FAB it should always be a plus)